libfuse/mount_test.go

```diff
@@ -2759,11 +2759,19 @@ func TestSimpleCRNoConflict(t *testing.T) {
 		t.Skip("Old FUSE protocol")
 	}
 
-	// both users read the root dir first
 	root1 := path.Join(mnt1.Dir, PrivateName, "user1,user2")
 	root2 := path.Join(mnt2.Dir, PrivateName, "user1,user2")
-	checkDir(t, root1, map[string]fileInfoCheck{})
-	checkDir(t, root2, map[string]fileInfoCheck{})
+	// both users should mutate the dir first
```

Comment:
Why is it required for both users to mutate?  Shouldn't it be enough for one user to mutate, and the other user to read it?  (I don't mind this way, I'm just making sure I understand what's going on.)